src/index.ts

```diff
@@ -295,6 +324,13 @@ export default class NewRelicLambdaLayerPlugin {
       environment.NEW_RELIC_SERVERLESS_MODE_ENABLED = "true";
     }
 
+    if (this.config.enableExtension) {
```

Comment:
We have a `logsEnabled` config option. Should this be `extensionEnabled` to be consistent?

Reply:
My thought was that that we have `enableIntegration`, so I was basing it off that--particularly since the two options could be used together.

Reply:
I could see the config option name go either way, since we have both `enable[Option]` and `[option]Enabled`. I lean more toward `enableExtension` because its most common use will be alongside `enableIntegration`, and there's a functional similarity in that each option is a method of delivering telemetry.